kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/richmessaging/views/KmFlowLayout.java

```diff
@@ -0,0 +1,62 @@
+package com.applozic.mobicomkit.uiwidgets.conversation.richmessaging.views;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.ViewGroup;
+
+public abstract class KmFlowLayout extends ViewGroup {
+
+    protected int line_height_space;
```

Comment:
some places we have _ and other places camelCase, is there any specific reason for using _ here?

Reply:
Will change it to camelCase